addons/payment_xendit/static/src/js/payment_form.js

```diff
@@ -200,6 +204,10 @@ paymentForm.include({
             card_exp_month: inputs.month.value,
             card_exp_year: inputs.year.value,
             card_cvn: inputs.cvn.value,
+            card_holder_email: inputs.email.value,
+            card_holder_first_name: inputs.first_name.value,
+            card_holder_last_name: inputs.last_name.value,
+            card_holder_phone_number: inputs.phone.value
```

Comment:
This will display the traceback dialog if the view template hasn't been updated yet. You should assume that `inputs.email` and others can be `undefined`.